runner.rb

```diff
@@ -1,42 +1,70 @@
-require_relative 'account'
-require_relative 'user'
+require_relative 'bank'
 
-def make_account(name)
-  Account.new({ owner: name })
-end
+# SauceBank runner class
+class Runner
+  attr_accessor :customer
+  def initialize
+    @bank = Bank.new
```

Comment:
You should inject your dependencies into your classes.  Your runner should be responsible for telling each class when to execute what method, but Bank should be initialized outside of the runner.